testdata/test-catalog.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterCatalog
+metadata:
+  finalizers:
+  - olm.operatorframework.io/delete-server-cache
```

Comment:
Not sure this is needed...
Shouldn't the finalizers be added by the controller?